examples/test/src/test/java/org/parceler/ParcelTest.java

```diff
@@ -130,6 +132,13 @@ public void testIntArrayFactoryOrder() {
         assertArrayEquals(new int[]{1, 2, 3, 4, 5}, unwrapped.getArr());
     }
 
+    @Test
+    public void testParcelableClassWithoutAnnotation() {
```

Comment:
No reason, it was just a native Parcelable. But I'd be willing to change if you want to.